test/functorch/test_control_flow.py

```diff
@@ -1251,8 +1309,163 @@ def mul(x: torch.Tensor, y: torch.Tensor):
             )
         self.assertEqual(cumsum1, cumsum_exp)
 
+    @requires_cuda
+    @parametrize("reverse", [False, True])
+    @parametrize("compile_mode", ["none", "eager", "compile", "compile_dynamic_shape"])
+    @parametrize("device", [torch.device("cpu"), torch.device("cuda")])
+    # Skipping the scan tests, as scan currently does not support inductor
+    @decorateIf(
+        unittest.skip,
+        lambda params: params["compile_mode"] != "eager"
+        and params["compile_mode"] != "none",
```

Comment:
Instead of skipping, maybe we should just remove "compile", "compile_dynamic_shape" from compile_mode list? Same comments for the rest of the tests.

Reply:
Yeah, my thought on this was that we keep the compilation modes and skip it for the moment, because we will hopefully soon provide an implementation for that and then we can just simply remove the decorator. However, I removed it for the moment to have a cleaner testfile. I left the if-clause though with a `TODO` in the test as a reminder.